### PR TITLE
Modified variable values from mx960/17.X to .* 

### PR DESCRIPTION
**Pull request by gvarao**
Addressed issues
- #122. Sane default for Software_Version_Value in software-version.rule
- #132. Rule check-task-memory-usage has wrong WHEN condition 
- #135. Typo in microburst rule

### CODE REVIEW

**Comment by ayushgithub** [comment]
jenkins please retry a build

File: juniper_official/Chassis/system-virtual-memory.yml [hunk 1]
```diff
@@ -11,10 +11,10 @@ systemVirtualMemoryView:
     system-vmstat-interrupts: _VMStatInterruptRatesTable
```
[the remark this answers is not in the document]
Comment (gvarao) [reply juniper_official/Chassis/system-virtual-memory.yml]: Please check now

**Comment by ayushgithub** [comment]
jenkins please retry a build

**Comment by ayushgithub** [comment]
jenkins please retry a build

**Comment by ayushgithub** [comment]
jenkins please retry a build

**Pull request closed without merging**